prototypes reflections (#46)
SW705E19/report: sections/discussion/ourprocessreflections.tex

    \label{fig:formal-reviews}
\end{figure}

\subsubsection{Prototypes}

> In the beginning of the semester we created multiple prototypes for the entire system so that everyone had the same understanding about how the design should be.

Reviewer: ```suggestion
In the beginning of the semester we created multiple prototypes for the entire system so that everyone had the same understanding of how the design should be.
```

> Some of these prototypes has been described in \autoref{sec:prototypes-design}.

Reviewer: ```suggestion
Some of these prototypes have been described in \autoref{sec:prototypes-design}.
```

\\\\

> When the development of the prototypes started we had a two people assigned the task of creating hi-fi prototypes, and afterwards people reviewed the prototypes and gave feedback and asked for changes.

Reviewer: ```suggestion
When the development of the prototypes started we had two people assigned to the task of creating high fidelity prototypes, and afterwards people reviewed the prototypes and gave feedback and asked for changes.
```

> Since the prototypes were high fidelity it was more difficult for reviewers to ask for big changes since it would take a lot of time to change.

Reviewer 2: Skal vi forklare hvad high fidelity er?

> This resulted in there never really being an overall discussion of how the design should be.

Reviewer: Ved ikke om jeg synes der ikke var en overall discussion, men ved ikke lige hvordan det ellers skal formulers. Der var vel en form for initial discussion, men det gik ikke så meget videre der fra???? 🤔

> A solution to this problem could be that the group developed lo-fi paper prototypes together. 

Reviewer: ```suggestion
A solution to this problem could be that the group developed low fidelity paper prototypes together. 
```

> A solution to this problem could be that the group developed low fidelity paper prototypes together. 

Reviewer 2: Igen, ved læseren godt hvad en low fidelity prototype er? Ved ikke om det er noget vi skal forklare i det tidligere prototype afsnit eller om det er ligegyldigt?

> This would give the people assigned the task of creating hi-fi prototypes something to develop them from, and would also lessen the amount of changes they had to do from the feedback.

Reviewer: ```suggestion
This would give the people assigned the task of creating high fidelity prototypes something to develop them from, and would also lessen the amount of changes they had to do from the feedback.
```

\\\\
When we started developing the application the prototypes were used a lot.

> These were used for inspiration for every task, but later in the process they started to get used less often and people did not always use them.

Reviewer: ```suggestion
These were used for inspiration for every task, but later in the process they started to get used less often and lost a bit of their importance.
```

> A reason for this can also be that you're able to see the design pattern from other pages, which can give you a good idea how to solve the tasks.

Reviewer: forstår ikke rigtigt den her sætning 🤔

> A reason for this can also be that the developers were able to see the design pattern from other pages, which could give them a good idea of how to solve the tasks.

Reviewer 2: ```suggestion
A reason for this could be that the developers were able to see the design pattern from other implemented pages, which could give them a good idea of how to solve the tasks.
```
